windows/add_to_path.tw2

```diff
@@ -9,26 +9,26 @@ To make Exercism usable on your computer, we need to move it to the right folder
 In the Command Prompt window, copy & paste:
 
 ```
-start %LOCALAPPDATA%\Microsoft\WindowsApps
+start "%LOCALAPPDATA%\Microsoft\WindowsApps"
 ```
 
 and hit the 'Enter' key.
 
 ### Verify
 
-A new window, showing a folder, will show up. This is where we need to place the `exercism` file. It will look something like this, but it might not be empty for you.
+A new window, showing a folder, will show up. This is where we need to place the `Exercism` file. It will look something like this, but it might not be empty for you.
```

Comment:
We need a clear disambiguation between Exercism and exercism:

- Exercism is the project name
- `exercism` is the command-line tool (technically `exercism.exe` but most people have extensions hidden)

In this case we're talking about the tool, not the Project.